bin/yi_appium_caps_util

```diff
@@ -29,8 +30,12 @@ end.parse!
 raise "You should pass -u or -c PLATFORM as an input argument. Type -h for help" if options[:update] == nil and options[:create_caps] == nil
 
 if options[:update]
-  if options[:file]
-    YiAppiumCapsUtil.update(options[:file])
+  if options[:file] && options[:platform]
+    YiAppiumCapsUtil.update(caps_file_name: options[:file], platformName_value: options[:platform])
```

Comment:
I dont think you would need all those ifelses, just `YiAppiumCapsUtil.update(caps_file_name: options[:file], platformName_value: options[:platform])` and in `update` you are already handling if the platformName is nill, right?